src/client/sandbox/event/simulator.js

```diff
@@ -16,6 +16,16 @@ const POINTER_EVENT_BUTTON = {
     rightButton: 2
 };
 
+// NOTE: event.detail !=0 in some Mouse Events (User Interface Events).
+// See https://www.w3.org/TR/uievents/#events-mouseevents
+// https://github.com/DevExpress/testcafe/issues/2232
```

Comment:
let's remove this comment with the issue. It's just a particular case. You link to `w3.org` is enough

Reply:
Done.